apps/extension/src/ui/apps/dashboard/routes/Settings/Mnemonics/MnemonicBackupModal/MnemonicBackupModalBase.tsx

```diff
@@ -0,0 +1,24 @@
+import { ModalDialog } from "talisman-ui"
+
+import { useMnemonicBackupModal } from "./context"
+
+export const MnemonicBackupModalBase = ({
+  children,
+  title,
+  className = "",
+}: {
+  children: React.ReactNode
+  title?: string
+  className?: string
+}) => {
+  const { close } = useMnemonicBackupModal()
+  return (
+    <ModalDialog
+      className={`w-auto p-2 ${className}`}
```

Comment:
when a component's classname needs to be overriden, need to use `className={classNames("w-auto p2", className)}` otherwise overrides for width and padding could be inconsistent. 
They'd be applied based on the order they are defined in the .css file generated by tailwind, on which we have no control.

Reply:
looks like this one got forgotten :)